src/it/MPMD-379-JDK21/invoker.properties

```diff
@@ -0,0 +1,28 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+invoker.java.version = 1.8+
+
+# available toolchains under linux:
+# https://github.com/apache/infrastructure-p6/blob/production/modules/build_nodes/files/toolchains.xml
+
+# the jdk toolchain "21:openjdk" is selected in pom.xml
+invoker.toolchain.jdk.version = 21
+invoker.toolchain.jdk.vendor = openjdk
```

Comment:
Really only `openjdk`?

Reply:
This is more like a smoke test (small project for java 21 that is compiled and analyzed), so any vendor will do, as long it is java 21. It's not testing compatibility with a specific vendor, but just compatibility with java 21. OpenJDK seems to be the tool chain, that is available on apache's jenkins instance.

Reply:
Then there is no need to restrict the vendor if any vendor will do. I guess a lot of people will also use Zulu which will not run here.

Reply:
Removed the vendor restriction.